libs/blocks/quiz/quiz.css

```diff
@@ -120,7 +120,7 @@ html[dir="rtl"] .quiz-option-icon {
   font-size: var(--type-body-xs-size);
   line-height: var(--type-body-xs-lh);
   margin: 0;
-  text-align: center;
+  text-align: left;
```

Comment:
Does this still work on RTL?

Reply:
Totally agree with @mokimo. I see a few other RTL issues on that page. If you support RTL use-cases, it might be helpful to open an RTL-specific story and go through all the potential improvements.
<img width="361" alt="Screenshot 2023-12-13 at 15 11 36" src="https://github.com/adobecom/milo/assets/11267498/77611b04-a3db-41ba-8f94-0dc7a02b706e">

Reply:
@mokimo Thanks for the feedback, I will update it
@overmyheadandbody Thanks for your feedback, I logged https://jira.corp.adobe.com/browse/MWPW-140478 to address all RTL issues